src/axom/slam/OrderedSet.hpp

```diff
@@ -371,7 +421,10 @@ struct OrderedSet : public Set<PosType, ElemType>,
    * An index pos is valid when \f$ 0 \le pos < size() \f$
    * \return true if the position is valid, false otherwise
    */
-  bool isValidIndex(PositionType pos) const { return pos >= 0 && pos < size(); }
+  inline bool isValidIndex(PositionType pos) const
+  {
+    return pos >= 0 && pos < size();
```

Comment:
Should the stride affect this?

Reply:
No. The _indices_ in an `OrderedSet` go from `0` to `size-1`. 
The _elements_ of the set have strides offsets and indirections.